huaweicloud/services/acceptance/dataarts/resource_huaweicloud_dataarts_architecture_reviewer_test.go

```diff
@@ -0,0 +1,131 @@
+	var (
+		region  = acceptance.HW_REGION_NAME
+		httpUrl = "v2/{project_id}/design/approvals/users?approver_name={id}"
+		product = "dataarts"
+	)
+
+	client, err := cfg.NewServiceClient(product, region)
+	if err != nil {
+		return nil, fmt.Errorf("error creating DataArts Studio client: %s", err)
+	}
+
+	getPath := client.Endpoint + httpUrl
+	getPath = strings.ReplaceAll(getPath, "{project_id}", client.ProjectID)
+	getPath = strings.ReplaceAll(getPath, "{id}", state.Primary.Attributes["user_name"])
+
+	getOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		MoreHeaders:      map[string]string{"workspace": state.Primary.Attributes["workspace_id"]},
+	}
+
+	getResp, err := client.Request("GET", getPath, &getOpt)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving DataArts Architecture reviewer: %s", err)
+	}
+
+	getRespBody, err := utils.FlattenResponse(getResp)
+	if err != nil {
+		return nil, err
+	}
+
+	jsonPaths := "data.value.records"
+	reviewers := utils.PathSearch(jsonPaths, getRespBody, make([]interface{}, 0)).([]interface{})
+	if len(reviewers) > 0 {
+		return reviewers, nil
+	}
+
+	return nil, golangsdk.ErrDefault404{}
+}
+
+func TestAccResourceArchitectureReviewer_basic(t *testing.T) {
+	var (
+		obj          interface{}
+		resourceName = "huaweicloud_dataarts_architecture_reviewer.test"
+	)
+	workspaceId := acceptance.HW_DATAARTS_WORKSPACE_ID
+	userId := acceptance.HW_DATAARTS_ARCHITECTURE_ID
+	userName := acceptance.HW_DATAARTS_ARCHITECTURE_NAME
+
+	rc := acceptance.InitResourceCheck(
+		resourceName,
+		&obj,
+		getArchitectureReviewerResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckDataArtsWorkSpaceID(t)
+			acceptance.TestAccPreCheckDataArtsArchitectureReviewer(t)
+		},
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccArchitectureReviewer_basic(),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttr(resourceName, "workspace_id", workspaceId),
+					resource.TestCheckResourceAttr(resourceName, "user_id", userId),
+					resource.TestCheckResourceAttr(resourceName, "user_name", userName),
+					resource.TestCheckResourceAttr(resourceName, "email", ""),
+					resource.TestCheckResourceAttr(resourceName, "phone_number", ""),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: testAccResourceArchitectureReviewerImportFunc(resourceName),
+			},
+		},
+	})
+}
+
+func testAccArchitectureReviewer_basic() string {
+	return fmt.Sprintf(`
+resource "huaweicloud_dataarts_architecture_reviewer" "test" {
+  workspace_id = "%s"
+  user_id      = "%s"
+  user_name    = "%s"
+  email        = ""
+  phone_number = ""
```

Comment:
you can use test value here
```suggestion
  email        = "test@example.com"
  phone_number = "12345678901"
```

Reply:
down